src/org/rascalmpl/values/parsetrees/TreeAdapter.java

```diff
@@ -44,9 +44,10 @@ public class TreeAdapter {
 	public static final String NORMAL = "Normal";
 	public static final String TYPE = "Type";
 	public static final String IDENTIFIER = "Identifier";
-	public static final String VARIABLE = "Variable";
-	public static final String CONSTANT = "Constant";
-	public static final String COMMENT = "Comment";
+	public static final String VARIABLE = "variable";
+	public static final String CONSTANT = "constant";
+	public static final String COMMENT = "comment";
+	public static final String COMMENT_LEGACY = "Comment";
```

Comment:
why only keep around support for `COMMENT_LEGACY` and not `CONSTANT_LEGACY`? and should we see if there are any users in java of these globals? and make sure that they are also updated? or was this premature to make these categories constants?

Reply:
I checked, there were no uses of `VARIABLE` and `CONSTANT`.